inbm-lib/inbm_lib/trtl.py

```diff
@@ -403,17 +386,16 @@ def image_remove_by_id(self, image_id: str, force: bool = False) -> Tuple[str, O
         @param force: Force image to be removed even if it has an active container
         @return: Result, error message, error code
         """
-        logger.debug("Trtl.imageRemoveByID(" + image_id + ")")
+        logger.debug("Trtl.ImageRemoveAll(" + image_id + ")")
         out, err, code = self.runner.run(self._boilerplate(
-            "imageRemoveByID") + " -id=" + image_id + " -f=" + str(force))
+            "ImageRemoveAll") + " -in=" + image_id + " -f=" + str(force))
```

Comment:
Here we are passing in an image_id to an instanceName parameter. Is this correct? It doesn't look right.

Reply:
image_id is not an id, it is a Name of Image. command ImageRemoveAll takes the image name as a parameter, based on that we changed it.
Please refer https://github.com/intel/intel-inb-manageability/blob/develop/inbm/trtl/README.md

Reply:
@Gsirisha12345 please have image_remove_by_id remove by id and add a different function image_remove_by_name for removing by name. Otherwise this is confusing for later maintainers.